SocialCareCaseViewerApi.Tests/V1/Gateways/DatabaseGateway.Tests.cs

```diff
@@ -1197,6 +1197,27 @@ public void UpdatePersonSetsCorrectValuesForPersonEntity()
             person.Title.Should().Be(request.Title);
         }
 
+        [Test]
+        public void PatchPersonSetsCorrectValuesForPersonEntity()
```

Comment:
Would be good to have test coverage for all properties in person entity, including all related entities that can be updated. Danger here is that it's possible to remove most of the gateway code and tests would still pass. So we would have records in database, update request and then expected result after patch. It's a lot of properties to deal with, but would improve safety a great deal.

Reply:
Done